ibm_cloud_sdk_core/utils.py

```diff
@@ -16,14 +16,30 @@
 # from ibm_cloud_sdk_core.authenticators import Authenticator
 import datetime
 import json as json_import
+import ssl
 from os import getenv, environ, getcwd
 from os.path import isfile, join, expanduser
 from typing import List, Union
 from urllib.parse import urlparse, parse_qs
 
+from requests.adapters import HTTPAdapter
+
 import dateutil.parser as date_parser
 
 
+class SSLHTTPAdapter(HTTPAdapter):
+    """Wraps the original HTTP adapter and adds additional SSL context.
+    """
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    #pylint: disable=arguments-differ
+    def init_poolmanager(self, connections, maxsize, block):
+        """Extends the parent's method by adding minimum SSL version to the args.
+        """
+        super().init_poolmanager(connections, maxsize, block, ssl_minimum_version=ssl.TLSVersion.TLSv1_2)
```

Comment:
Yeah that's right, we need to use `ssl_context`! I've already made this change, but haven't pushed it yet, because I want to do some further testing to make sure everything works fine.
By the way, I found it here: https://github.com/urllib3/urllib3/blob/main/src/urllib3/poolmanager.py#L53, but it doesn't work, because this is the main branch (future 2.0 release) and this option is missing from the current release branch (1.26.x).